Blimp/mode_velocity.cpp

```diff
@@ -0,0 +1,33 @@
+#include "Blimp.h"
+/*
+ * Init and run calls for velocity flight mode
+ */
+
+// Runs the main velocity controller
+void ModeVelocity::run()
+{
+    Vector3f target_vel;
+    target_vel.x = channel_front->get_control_in() / float(RC_SCALE) * g.max_vel_xy;
+    target_vel.y = channel_right->get_control_in() / float(RC_SCALE) * g.max_vel_xy;
+    blimp.rotate_BF_to_NE(target_vel.x, target_vel.y);
+    target_vel.z = channel_down->get_control_in()  / float(RC_SCALE) * g.max_vel_z;
+    float target_vel_yaw = channel_yaw->get_control_in() / float(RC_SCALE) * g.max_vel_yaw;
+
+    Vector2f actuator = blimp.pid_vel_xy.update_all(target_vel, blimp.vel_ned_filtd, {0,0,0});
+    blimp.rotate_NE_to_BF(actuator.x, actuator.y);
+    float act_down = blimp.pid_vel_z.update_all(target_vel.z, blimp.vel_ned_filtd.z);
+    float act_yaw = blimp.pid_vel_yaw.update_all(target_vel_yaw, blimp.vel_yaw_filtd);
+
+    if(!(blimp.g.dis_mask & (1<<(2-1)))){
+    motors->front_out = actuator.x;
+    } if(!(blimp.g.dis_mask & (1<<(1-1)))){
```

Comment:
each if should start on a new line (this would be nicer as a table actually)

Reply:
here is an example as a table:
```
    const struct {
        float &axis;
        float &mout;
        uint8_t dis_mask;
    } disable_table[] = {
        { actuator.y, motors->right_out, 1 },
        { actuator.x, motors->front_out, 2 },
        { act_down,   motors->down_out,  3 },
        { act_yaw,    motors->yaw_out,   4 },
    };
    for (const auto &t : disable_table) {
        if ((blimp.g.dis_mask & (1<<(t.dis_mask-1))) == 0) {
            t.mout = t.axis;
        }
    }
```

Reply:
Thanks. Would I then need to have this block at each spot where I'm currently using if statements? 
Seems like it'd be more difficult to read than the four if statements?
I was actually considering removing the dis_mask entirely as tuning wasn't too bad once I got all the directions going the right way. Should I do that instead?